.travis.yml

```diff
@@ -9,22 +9,34 @@ matrix:
     - os: linux
       dist: xenial
       python: 2.7
-      env: CPLUS_INCLUDE_PATH=/usr/include/gdal C_INCLUDE_PATH=/usr/include/gdal
     - os: linux
       dist: xenial
       python: 3.7
-      env: CPLUS_INCLUDE_PATH=/usr/include/gdal C_INCLUDE_PATH=/usr/include/gdal
     - os: osx
       language: generic
 
+
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-add-repository -y ppa:ubuntugis/ppa; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install gdal-bin libfftw3-dev libgdal-dev geographiclib-tools libgeographic-dev libgsl-dev; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then HOMEBREW_NO_AUTO_UPDATE=1 brew install geographiclib fftw gsl; fi
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      sudo apt-add-repository -y ppa:ubuntugis/ppa
+      sudo apt-get -qq update
+      sudo apt-get install gdal-bin libfftw3-dev libgdal-dev geographiclib-tools libgeographic-dev libgsl-dev
```

Comment:
Was `libgsl-dev` left here intentionally, or is it a mistake? I see it was removed from the README and from the macos dependencies a few lines below.

Reply:
it's a mistake, gsl is not necessary

Reply:
I just removed it